### DISCUSSION TITLE
Charge top-up based on calendar event

### DISCUSSION THREAD

**Discussion by eighteentee** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#726] [Ideas]
I take my Mac out into the field twice a week, away from power. The rest of the time it sits on my desk pinned to 80% charge through Aldente. Would it be a good idea for Aldente to, say, be able to look at calendar events and based on criteria there, charge the device up to 100% before taking the device away from power? This would be extremely handy and really be a set-it-and-forget-it solution for remembering to top-up the device before leaving. Just a thought...

**Comment by remicarreiro** [comment]
Glad i'm not the only one who's been wondering about this. I use my laptop on photoshoots and would love to have it top-up before a calendar event for that shoot if possible.
